metal-sensing/README.md

```diff
@@ -0,0 +1,26 @@
+# Package: Metal-sensing
+
+http://parts.igem.org/Collections/Metal
+
+### Summary:
+
+- 7 parts
+    - CDS: 1
+    - engineered_region: 3
+    - promoter: 1
+    - ribosome_entry_site: 1
+    - terminator: 1
+- 1 vectors
+- 0 samples for distribution _<span style="color:red">8 parts not included</span>_ _<span style="color:red">No samples planned to be built for distribution</span>_
```

Comment:
Something is going wrong here, since the metal-sensing parts aren't being included in the distribution plan. It certainly looks like it should be, so I'll check it out...

Reply:
I've figured out the problem @dkiga : the build plans were ignoring non-combinatorial entries on the libraries and composites sheet. Your package is the first that happens to prominently use those (and thus revealed the bug). I have a fix pending on https://github.com/iGEM-Engineering/iGEM-distribution/pull/198

Reply:
Fix is implemented.